slackmessage/send-to-channel-slack.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: tekton.dev/v1alpha1
+kind: Task
+metadata:
+  name: send-to-channel-slack
+spec:
+  inputs:
+    params:
+    - name: token-secret
+      type: string
+      description: seret name of the slack app access token (key is token)
+    - name: channel
+      type: string
+      description: channel id or channel name
+    - name: message
+      type: string
+      description: plain text message
+  steps:
+  - name: post
+    image: curlimages/curl:7.68.0
+    command: ["/bin/sh"]
```

Comment:
You could use `script` there instead :angel:

Reply:
Thanks for review.  I made the change.  It became much easier and cleaner.